definitions/ext-cisco_ucs/golden_metrics.yml

```diff
@@ -21,3 +21,12 @@ transmitErrors:
     select: sum(kentik.snmp.ifOutErrors)
     from: Metric
     where: "provider = 'kentik-ucs'"
+
+uptime:
+  title: Uptime
+  unit: SECONDS
+  query:
+    select: latest(kentik.snmp.Uptime)/100
+    from: Metric
+    where: "provider = 'kentik-ucs'"
+    eventId: entity.guid
```

Comment:
> another question: what is happening to the [device_ip](https://github.com/newrelic/entity-definitions/blob/264efca3eb69579c311a52ff84a6be3c19855273/definitions/ext-cisco_ucs/summary_metrics.yml#L1-L5) tag that is present on almost all network entity summary metric definitions today?

This is going to stay. 
The plan, is to make all the NRDB metrics be migrated to golden-metrics. Whereas the tags will remain like they are.

Reply:
It will highly depend on each case, But in general, in this PR I tried to keep it. Do you foresee any problem or concern here?

Reply:
@naxhh might have a better idea if this will conflict with the GM definitions in any way. I am not positive why the `eventId` field is necessary for SM, but my assumption is that if we're moving these to GM this field can be dropped.